ui/admin/app/components/form/role/manage-scopes/index.js

```diff
@@ -0,0 +1,48 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import {
+  GRANT_SCOPE_THIS,
+  GRANT_SCOPE_CHILDREN,
+  GRANT_SCOPE_DESCENDANTS,
+} from 'api/models/role';
+
+export default class FormRoleManageScopesIndexComponent extends Component {
+  // =attributes
+
+  KEYWORDS = {
```

Comment:
Do we normally use all caps for the object/key?

Reply:
originally i was using lower case but I got error in the template when trying to access the "this" key in the "keywords" object. I think the another option would be to rename keys to "keyThis" "keyChildren" "keyDescendants".

Reply:
ahh yeah I would probably do that, it seems very uncharacteristic to use all caps for this